packages/account/src/Sections/Verification/ProofOfOwnership/Card.jsx

```diff
@@ -56,6 +62,8 @@ const Card = ({ card, handleChange, handleBlur, values, setFieldValue, index, er
                     error={error}
                     validateField={validateField}
                     updateErrors={updateErrors}
+                    show_button={show_button}
```

Comment:
which button?

Reply:
Hello @ali-hosseini-fs, Sure, I'll change this to show_browse_button